client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabs.tsx

```diff
@@ -41,7 +41,7 @@ const ContactsTabs = (props : Props) => {
                 <Tab classes={tabClasses} icon={<FontAwesomeIcon color={active} icon={faUserEdit} />} label={mutualContactsLabel} />
                 <Tab classes={tabClasses} icon={<GroupOutlined color='action' />} label={familyMembersLabel} />
                 <Tab classes={tabClasses} icon={<CallMerge color='action' />} label={groupedInvestigationsLabel} />
-                <Tab classes={tabClasses} icon={<AccountBalance color='action' />} label={contactBankLabel} />
+                {/* <Tab classes={tabClasses} icon={<AccountBalance color='action' />} label={contactBankLabel} /> */}
```

Comment:
please remove comment

Reply:
The comment are so that we can add it back after